spec/models/comment_spec.rb

```diff
@@ -2,6 +2,32 @@
 
 require 'rails_helper'
 
-RSpec.describe Comment, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+RSpec.describe CommentsController, type: :controller do
```

Comment:
rspec for CommentsController should be write in file `spec/controllers/comments_controller_spec.rb`